.github/workflows/build-test-package.yml

```diff
@@ -13,17 +13,17 @@ jobs:
           - os: ubuntu-18.04
             c-compiler: "gcc"
             cxx-compiler: "g++"
-            itk-git-tag: "v5.1.1"
+            itk-git-tag: "v5.1.2"
```

Comment:
Do I understand correctly that this PR would upgrade ITKElastix from ITK 5.1.1 to 5.1.2? If so, could you please mention this in the commit message (or subject)?

Reply:
Correct -- I added more information in the subject.